runtimes/pythonrt/pythonrt.go

```
@@ -647,7 +620,7 @@ func (py *pySvc) Call(ctx context.Context, v sdktypes.Value, args []sdktypes.Val
 				py.log.Log(pyLevelToZap(r.level), r.message)
 				close(r.doneChannel)
 			case p := <-py.channels.print:
-				py.cbs.Print(ctx, py.runID, p.message)
+				_ = py.cbs.Print(ctx, py.runID, p.message)
```

Don't ignore the error

---

done